fluent-plugin-google-cloud.gemspec

```diff
@@ -19,19 +19,19 @@ eos
   gem.test_files    = gem.files.grep(/^(test)/)
   gem.require_paths = ['lib']
 
-  gem.add_runtime_dependency 'fluentd', '~> 1.2', '>= 1.2.5'
-  gem.add_runtime_dependency 'googleapis-common-protos', '~> 1.3'
-  gem.add_runtime_dependency 'google-api-client', '~> 0.17'
-  gem.add_runtime_dependency 'google-cloud-logging', '~> 1.3', '>= 1.3.2'
-  gem.add_runtime_dependency 'googleauth', '~> 0.6'
-  gem.add_runtime_dependency 'grpc', '~> 1.0'
-  gem.add_runtime_dependency 'json', '~> 2.1'
-  gem.add_runtime_dependency 'google-protobuf', '~> 3.6', '>= 3.6.1'
+  gem.add_runtime_dependency 'fluentd', '1.2.5'
+  gem.add_runtime_dependency 'googleapis-common-protos', '1.3.7'
+  gem.add_runtime_dependency 'googleauth', '0.6.6'
```

Comment:
Did we have to sort this in the same commit that changed the versions? This makes it a lot harder to find the actual diffs...

Reply:
This file is small enough so I did not realize it would cause much confusion.

Added two commits to split them up.

Reply:
To see version diff:
https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/pull/262/files/0fff479641e19fec43ed0b9f2e51fe99de33dd08

To see order diff:
https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/pull/262/commits/f569fae5d8a366b6cf8ce815d013bf88145e2363

Reply:
Thanks, this helped.